web/src/components/button.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'gatsby';
+
+export default function Button({
+    isPrimary,
+    isPrimaryDark,
+    isSecondary,
+    isSecondaryDark,
+}) {
+    const primaryStyle =
+        'bg-black text-white hover:bg-brand-1 visited:bg-gray-300';
+    const primaryDarkStyle =
+        'bg-white text-black hover:bg-gray-300 visited:bg-brand-1';
+    const secondaryStyle =
+        'bg-white text-black hover:bg-brand-1 hover:text-brand-1 visited:text-gray-300';
+    const secondaryDarkStyle =
+        'bg-brand-1 text-white hover:text-gray-300 visited:text-brand-1';
+
+    const finalStyles = `border-2 border-current h-9 rounded-bl-lg uppercase w-48 ${
```

Comment:
I don't think you need `w-48`. The button's width should be dictated by content and padding.

Same for `h-9`.

Also, should there be font size/weight, line-height, text-transform, and letter spacing classes added here?

Reply:
- there isnt a class for content height
- if i remove the width, and add the padding, we have something like this: thoughts? i thought it would be remedied with display, but thats not doing it either
<img width="292" alt="Screen Shot 2021-05-27 at 3 28 06 PM" src="https://user-images.githubusercontent.com/61708936/119885696-26652480-bf00-11eb-89c9-c2181fe4818a.png">

Reply:
Adjusting the line height or bottom padding should solve.

You could try to add flex and the use justify and align to center the text but that might be more trouble than it's worth. If option 1 doesn't seem like it's going to work let me know and I'll take a closer look.

Reply:
@asasmith is has bottom padding?

Reply:
@asasmith flex didnt work either, i tried that before I sent the comment

Reply:
@kcbreuil I think this is a little challenging because you're rendering it on it's own instead of wrapping it in an element. I added `display: inline-block` and I think that helps.

Reply:
I've also added some padding because I don't think you have your most recent changes pushed.

![Screen Shot 2021-05-27 at 4 56 50 PM](https://user-images.githubusercontent.com/11489171/119896091-8b267c00-bf0c-11eb-8c56-9903aaa3f115.png)

Reply:
@asasmith see updated PR